stable/distribution/README.md

```diff
@@ -44,6 +44,16 @@ Once you have a new chart version, you can update your deployment with
 helm upgrade distribution jfrog/distribution
 ```
 
+If distribution was installed without providing a value to redis.password (a password was autogenerated), follow these instructions:
+1. Get the current password by running:
+```bash
+REDIS_PASSWORD=$(kubectl get secret -n <namespace> <myrelease>-redis-secret -o jsonpath="{.data.password}" | base64 --decode)
```

Comment:
Need also to extract postgresql password

Reply:
Done